test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -44,7 +44,7 @@ public void AddUserSecrets_With_An_Existing_Secret_File()
             var projectPath = UserSecretHelper.GetTempSecretProject(out userSecretsId);
```

Comment:
Test classes in this package do not following the [Engineering Guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-class-naming). Should be in the `Microsoft.Extensions.SecretManager` namespace (no `.Tests` suffix) and be named `...Test` (singular). Since you're touching many of these files, might be worth fixing now. If not, please file a low-priority Task.